spec/features/post_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'capybara/rspec'
+require 'capybara'
+require 'selenium-webdriver'
+require_relative '../../spec/spec_helper'
+require_relative '../../pages'
+require_relative '../../locators'
+
+feature 'OpenSourceCMS Posts test', type: :feature do
+  describe Pages::Posts do
+
+    before(:each) do
+      @session = Pages::Posts.new
```

Comment:
it is better to name such variables same as the page you are creating, for example - posts_page = Pages::Posts.new. This way it will be more readable and logical. Also its better to use local variables instead of instance variable or use let declarations if possible